### ISSUE TITLE
[BUG] Bicep contains cosmosdb but doesn't deploy it

### ISSUE THREAD

**Issue by sander1095** [Azure-Samples/dab-azure-cosmos-db-nosql-quickstart#1]
### Description

The bicep contains definitions for both Azure SQL and CosmosDB Nosql.

However, inspecting files like `infra/app/database.bicep` shows that this only deploys Azure SQL. In fact, I can not find any usage of the CosmosDB anywhere in bicep.

### Expected behavior

I would expect this repo to have NOTHING to do with Azure SQL. Instead, only cosmosdb should be used. The other example repo for dab-sql should use azure sql instead.

**Comment by seesharprun** [comment]
@sander1095, you're right. We're going to get a fix live for this. It was copied from the Azure SQL variant of the repo (azure-samples/dab-azure-sql-quickstart) and this must have been missed.

**Comment by sander1095** [comment]
Hey @seesharprun , that's awesome to hear. I see there's already a draft PR.

Will this be merged/ready to deploy when cloned locally before the weekend? I got a livestream next week about DAB + GraphQL and I'd love to use this repo for some of that.

**Comment by seesharprun** [comment]
I don't know if I will have things finalized this week.

We do have other guides at https://aka.ms/dab-docs that could help with your event.

We have guides and tutorials there.

For example, https://learn.microsoft.com/en-us/azure/data-api-builder/quickstart-nosql

And there's four more code samples.

**Comment by ryanspain** [comment]
Would love to see that PR merged in.

I was a little surprised by all the Infra-as-Code Bicep files that don't seem relevant to this same. E.g. cognitive services.

Is there an ETA on #2 being merged? Is there anything I can do to help?
